crates/sui-graphql-rpc/schema.graphql

```diff
@@ -0,0 +1,840 @@
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Open Questions
+# --------------
+# - Rules for nullabilit/non-nullability
+# - Pattern for multi-gets
+# - Encoding RPC errors
+# - `type` (vs `scalar`) for Move Types
+# - Pattern for upcasting and downcasting
```

Comment:
Proposal uses a pattern where types that can be cast between each other have `asX: X` fields (e.g. `MoveObject` has `asCoin: Coin`, `asStakedSui: StakedSui`, etc, which can fail, but also `asObject: Object` which will always succeed).